testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -310,18 +318,60 @@ public void KeepAliveClientRpc()
         m_TimeSinceLastKeepAlive = Time.time;
     }
 
+    private ulong[] m_TargetClient = new ulong[1] { 0 };
+    private ClientRpcParams m_ClientParams = new ClientRpcParams();
+
+    public delegate void KeepServerFromTimingOutDelegateHandler();
+
+    public KeepServerFromTimingOutDelegateHandler KeepServerFromTimingOut;
+
     [ServerRpc(RequireOwnership = false)]
+    //public void WriteTestResultsServerRpc(float result, ServerRpcParams receiveParams = default)
     public void WriteTestResultsServerRpc(float result, ServerRpcParams receiveParams = default)
     {
         var senderId = receiveParams.Receive.SenderClientId;
+        Debug.Log($"Server received result [{result}] from sender [{senderId}]");
         if (!m_TestResultsLocal.ContainsKey(senderId))
         {
             m_TestResultsLocal[senderId] = new List<float>();
         }
 
         m_TestResultsLocal[senderId].Add(result);
+
+
+        if (KeepServerFromTimingOut != null)
+        {
+            KeepServerFromTimingOut.Invoke();
+        }
+
+        // Now send the results received verification
+        m_TargetClient[0] = senderId;
+        m_ClientParams.Send.TargetClientIds = m_TargetClient;
```

Comment:
mmm why is this a field and not just a var? I don't see it used anywhere else (and looking at the logic, it shouldn't be used anywhere else, as its value is based on the current context)

Reply:
m_ClientParams I would say we could make that a local var.
m_TargetClient should stay a field and pre-allocated.

Reply:
done, please take a look at it now and see if it is good to go.

Reply:
I don't see any changes.
Noel: allocating an array of long of size 1 doesn't too big of a deal to me.
I'm nit picking here since there's a danger to having "global" right? m_ClientParams and m_Targetclient could be changed anywhere. If they are local to this context, it's self contained.

Reply:
bump